src/test/java/org/jenkinsci/plugins/workflow/steps/ToolStepRunTest.java

```diff
@@ -60,4 +73,69 @@
         r.assertLogContains("Apache Maven 3", r.assertBuildStatusSuccess(p.scheduleBuild2(0)));
     }
 
+    @Test public void toolWithSymbol() throws Exception {
+        File buildDirectory = new File(System.getProperty("buildDirectory", "target")); // TODO relative path
```

Comment:
:ant: either relativize to `r.jenkins.getRootDirectory()`, or use `@Rule public TemporaryFolder`.